src/components/Core/Nav/NavItemWithMenu.js

```diff
@@ -10,11 +10,12 @@ import { SvgSolidChevronDown } from '@/components/Svg'
 const StyledMenuItem = styled.div`
   cursor: pointer;
   font-size: 12px;
-  margin: 10px 0;
-  transition: color 0.2s ease;
-
+  padding: 0.5rem 1rem;
+  transition: 0.2s ease;
   &:hover {
```

Comment:
Let's add space between, it seems common in styled-components to have an extra linefeed here.